deepchem/data/tests/test_fastq_loader.py

```diff
@@ -0,0 +1,43 @@
+import os
+import unittest
+from deepchem.feat.molecule_featurizers import OneHotFeaturizer
+from deepchem.data.data_loader import FASTQLoader
+
+
+class TestFASTQLoader(unittest.TestCase):
+  """
+  Test FASTQLoader
+  """
+
+  def setUp(self):
+    super(TestFASTQLoader, self).setUp()
+    self.current_dir = os.path.abspath(os.curdir)
```

Comment:
Same error here. You're looking in the top level rather than the current file's directory